release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -40,6 +40,41 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-7-enhanced-disk-provisioning"]
+==== Enhanced disk provisioning for LUKS and RAID devices
+
+{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments:
+
+* Native Ignition support for LUKS disk encryption provides additional configurability for encrypted root filesystems, as well as support for encryption of additional data filesystems.
+* {op-system} now supports boot disk mirroring, except on s390x, providing redundancy in the case of disk failure.
+* {op-system} on s390x can be installed onto FBA DASD disks.
+
+[NOTE]
+====
+LUKS configuration can no longer be performed in the `/etc/clevis.json` file. Provisioning fails if `/etc/clevis.json` is specified in the machine config.
```

Comment:
It turns out that only new clusters can switch to native Ignition LUKS, and existing clusters should continue to use `clevis.json`.

Reply:
Not sure I am understanding right 😁  but let me know if this rewording makes sense:

```
[NOTE]
====
Native Ignition LUKS configuration can only be performed on new clusters. Existing clusters must continue to specify the `/etc/clevis.json` file in the machine config.
====
```
❓

Reply:
This is more words but hopefully conveys a fuller picture.  WDYT?

> On new clusters, LUKS configuration must use the native Ignition mechanism; provisioning will fail if the legacy `/etc/clevis.json` file is included in the machine config.  On clusters upgraded from {prod-name} 4.6 or earlier, LUKS can only be configured via `/etc/clevis.json`.

(I'm trying to avoid implying in either case that LUKS _must_ be specified, since it's an optional feature.)